### PR TITLE
fix(bc): form type support of >2.8

### PR DESCRIPTION
**Pull request by amine2z**
I am targetting this branch, because…

Closes #
Fixes #

## Changelog

```markdown
### Fixed
 - symfony 3 type support in AclMatrixType

```

## Subject
Support symfony3 form type naming

### CODE REVIEW

File: Form/Type/AclMatrixType.php [hunk 1]
```diff
@@ -33,10 +33,22 @@ public function buildForm(FormBuilderInterface $builder, array $options)
...
         $aclValueData = $options['acl_value'] instanceof UserInterface ? $options['acl_value']->getUsername() : $options['acl_value'];
 
-        $builder->add($aclValueType, 'hidden', array('data' => $aclValueData));
+        $builder->add(
+            $aclValueType,
+            // NEXT_MAJOR: remove when dropping Symfony <2.8 support
+            method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
+                ? 'Symfony\Component\Form\Extension\Core\Type\HiddenType'
+                : 'hidden',
+            array('data' => $aclValueData));
```
Comment (greg0ire) [inline Form/Type/AclMatrixType.php]: The last parenthesis should be on its own line

**Comment by greg0ire** [comment]
(Almost) perfectly executed. Can you fix StyleCI ?

**Comment by greg0ire** [comment]
Do you need help?

**Comment by greg0ire** [comment]
LGTM

**Comment by core23** [comment]
Thank you @amine2z

**Pull request merged**